objax/io/checkpoint.py

```diff
@@ -92,6 +92,8 @@ def restore(self, vc: VarCollection, idx: Optional[int] = None):
         if idx is None:
             all_ckpts = glob.glob(os.path.join(self.logdir, self.DIR_NAME, self.FILE_MATCH))
             if not all_ckpts:
+                if self.verbose:
+                    print('WARNING: Empty path provided; not restoring from any checkpoint.')
```

Comment:
Better error message would be: "No checkpoint found, skip restoring variables."

Reply:
Changed.